### ISSUE TITLE
Cannot override environment tag

### ISSUE THREAD

**Issue by blackjid** [getsentry/raven-ruby#646]
Hi, I'm trying to setup a rails project to report events in different environments using the tags attribute.

```
Raven.configure do |config|
  config.tags = { environment: 'staging' }
end
```

The thing is that this environment is really running the production environment on rails. `Rails.env = "production"`

This used to work, but not anymore, now all events have their environment tag as `production` regardless the value of the environment tag. In fact I remember reading this on the ruby sdk documentation, but I couldn't find it now..

Any ideas??

**Comment by blackjid** [comment]
well, even if I manually do something like this.

```
Raven.capture_message "My event",
  logger: 'logger',
  extra: {
    my_custom_variable: 'value'
  },
  tags: {
    environment: 'staging'
  }
```

The event is saved with the tag environment: `"production"`

**Comment by nateberkopec** [comment]
Hm, yeah the docs are misleading here. 

What you can do is:

```ruby
Raven.configure do |config|
  config.current_environment = "staging"
end
```

[Environment is not a tag, it's a core attribute of a Sentry event.](https://docs.sentry.io/clientdev/attributes/)

**Comment by blackjid** [comment]
thanks! @nateberkopec. 

I fact, I tried this and it worked... 

```ruby
Raven.capture_message "My event",
  logger: 'logger',
  extra: {
    my_custom_variable: 'value'
  },
  environment: 'staging'
```

I wonder if this changed at some point or it has always been this way?

**Comment by nateberkopec** [comment]
Sentry's server is actually super permissive and will often times "correct" a lot of the data that gets sent. My hunch is that we used to send environment as a tag, which got *corrected* on the server to be treated as a core attribute.

Once we started sending the core attribute correctly (https://github.com/getsentry/raven-ruby/commit/3d820eba4d7c367e4ca1e1568587577257ee542d) it made the documentation incorrect.
